guppylang/definition/custom.py

```diff
@@ -85,15 +92,16 @@ def compile_call(
     ) -> Sequence[Wire]:
         """Compiles a call to the function."""
         # Note: We have _compiled_ globals rather than `Globals` here,
-        # so we cannot use `self._get_signature()`.
+        # so we cannot use `self._get_signature()` to build a `CustomFunctionDef`.
```

Comment:
Having tweaked this doc slightly, I still have to say it'd be good to have some doc as to why we might ever do this without parsing it into a CustomFunctionDef first...